lib/Configuration.php

```diff
@@ -80,13 +80,11 @@ class Configuration
         'traffic' => array(
             'limit'      => 10,
             'header'     => null,
-            'dir'        => 'data',
             'exemptedIp' => null,
         ),
         'purge' => array(
             'limit'     => 300,
             'batchsize' => 10,
-            'dir'       => 'data',
```

Comment:
Required for backward-compatability when importing value to database, etc.

Reply:
I decided not to implement a migration of purge and traffic limiter migration into database, only server salt and only if that is in the default location (data):
- purge limiter has a life time of 300s, so it immediately purges on the first paste or comment creation after the upgrade, but on the DB that is a very cheap operation (we query the paste table with a where condition on the expiration timestamp)
- traffic limiter has a life time of 10s, so within 10s after the upgrade is completely empty/to be purged anyway.

Apart from mentioning this change in the CHANGELOG.md, we should point this change out in the upgrade instructions, i.e. "If you use the (default) Filesystem storage and have changed the `dir` for either purge or paste limiter, move these files into the same `dir` as configured in the `model_options`. If you use the Database storage, make sure that the `config` table contains the `salt` row from the file `data/salt.php` - an automatic migration is attempted, but may fail if you used a non-standard `PATH`."